README.md

```diff
@@ -10,7 +10,7 @@ helm dependency update ./
 ```
 - Then install the helm chart with devProfile=true to make sure that Kafka and Apicurio get set up in the cluster:
 ```script
-helm install sbomer-release ./ --set global.includeKafka=true --set global.includeApicurio=true --set global.includeApiGateway=true -n sbomer --create-namespace
+helm install sbomer-release ./ -n sbomer-test --create-namespace
```

Comment:
I tripped up here, for the time being we do need to call this `sbomer-release` but it should be more flexible in future, perhaps name the services in a more generic way but use the namespace for hosts ie `kafaka.svc.sbomer-dev.cluster.local` (i always forget what comes where!)